src/validation_workflow/api_test_cases.py

```diff
@@ -28,10 +28,10 @@ def test_cases(self) -> Any:
 
         # the test case parameters are formated as ['<request_url>',<success_status_code>,'<validate_string(optional)>']
         test_cases = [
-            ['https://localhost:9200/', 200, '"number" : "' + self.opensearch_image_version + '"'],
-            ['https://localhost:9200/_cat/plugins?v', 200, ''],
-            ['https://localhost:9200/_cat/health?v', 200, 'green'],
-            ['http://localhost:5601/api/status', 200, '"number":"' + self.opensearch_dashboards_image_version + '"']
+            ['https://localhost:9200/', 0, '"number" : "' + self.opensearch_image_version + '"'],
+            ['https://localhost:9200/_cat/plugins?v', 0, ''],
+            ['https://localhost:9200/_cat/health?v', 0, 'green'],
+            ['http://localhost:5601/api/status', 0, '"number":"' + self.opensearch_dashboards_image_version + '"']
```

Comment:
curl reponses do return https status codes as well https://stackoverflow.com/questions/2220301/how-to-evaluate-http-response-codes-from-bash-shell-script
Can we do that instead of 0 here?

Reply:
oops .. got it ,,, sure @gaiksaya . let me do that instead of using 0 ..... thanks for the info...

Reply:
@gaiksaya  .. I have it updated with status code 200 by using '-i' to get the HTTP response header in the Curl .. it looks working. thanks!! The --write-out or -w was difficut to implement because it only fetch the header, but I want to get header and content both. The '-i' looks good to use in Curl. thanks!!